index.html

```diff
@@ -57,6 +57,230 @@ <h3 id="social-heading">Let's Connect</h3>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="card-skills">
+                            <ul>
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn project-btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot4.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <h3 class="card-title">Multi-Post Stories</h3>
+                            <div class="card-bg-info">
+                                <p>Canopy</p>
+                                <ul>
+                                    <li>Back End Dev</li>
+                                    <li>2015</li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="card-text">
+                            <p>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="card-skills">
+                            <ul>
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn project-btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <!--About Section-->
+        <section id="about">
+            <div class="about-container">
+                <div class="about">
+                    <div>
+                        <h2>About Myself
+                        </h2>
+                    </div>
+                    <div>
+                        <p>
+                            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, don't hestiate to contact me.
+                        </p>
+                    </div>
+                    <div>
+                        <div>
+                            <h4 id="about-heading">Let's Connect</h4>
```

Comment:
- [ ] Please 🙏🏽 use the `a` tag instead of `hx` for the best practices of HTML & CSS because this is a link for the contact form.